### PR TITLE
feat(ext): new extension - Redirect to custom error page

### PR DESCRIPTION
**Pull request by BabooPan**
This extension allows users to define a custom error page with a specific status code from the origin response.

Refer to the solution from [Customize 403 error pages from Amazon CloudFront Origin with Lambda@Edge](https://aws.amazon.com/blogs/networking-and-content-delivery/customize-403-error-pages-from-amazon-cloudfront-origin-with-lambdaedge/)

### CODE REVIEW

**Comment by pahud** [comment]
圖片


run`yarn upgrade` and `npx projen` again.

**Comment by pahud** [comment]
LGTM

@guan840912 can you check this?

**Review by pahud — APPROVED** [review]

**Comment by pahud** [comment]
Let's ship it atm. Will confirm the solution ID and function name with @NingLu.

**Review by guan840912 — APPROVED** [review]

File: src/extensions.ts [hunk 1]
```diff
@@ -260,6 +260,26 @@ export class DefaultDirIndex extends Custom {
...
+export class RedirectCustomErrorPage extends Custom {
+  readonly lambdaFunction: lambda.Version;
+  constructor(scope: cdk.Construct, id: string) {
+
+    super(scope, id, {
+      runtime: lambda.Runtime.PYTHON_3_7,
+      handler: 'index.handler',
+      code: lambda.AssetCode.fromAsset(`${EXTENSION_ASSETS_PATH}/redirect-custom-error-page`),
+      eventType: cf.LambdaEdgeEventType.ORIGIN_RESPONSE,
+      solutionId: '',
```
Comment (guan840912) [inline src/extensions.ts:277]: @pahud solutionId need ?!

Comment (pahud) [reply src/extensions.ts:277]: We'll  confirm the solution ID with @NingLu in the next few days and I'll cut a seperate PR for it. @guan840912

**Comment by pahud** [comment]
I'll cut a PR to update the solution id:

SO8136	cf-customize-error-page

**Comment by pahud** [comment]
https://github.com/pahud/cdk-cloudfront-plus/pull/55

**Pull request merged**